cicd/42c-conf-examples/mapping-no-discovery/42c-conf.yaml

```diff
@@ -27,6 +24,8 @@
 # the platform at all.
```

Comment:
" The sample 'petstore.yaml' is not uploaded to the platform at all." --> " The sample 'petstore.yaml' is unmapped and thus is not uploaded to the platform at all."